docs/tutorials/_jellyfish-architecture.mdx

```diff
@@ -0,0 +1,63 @@
+:::info
+
+You can learn more about Jellyfish architecture in [Jellyfish
+docs](/). This section provides a
```

Comment:
update reference to docs

Reply:
But this is valid reference. It points to main page